content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration.rst

```diff
@@ -174,6 +177,55 @@ from the shipping carrier.
 .. seealso::
    :doc:`Invoice cost of shipping <setup_configuration/invoicing>`
 
+.. _inventory/shipping_receiving/shipping-route:
+
+Route on shipping method
+------------------------
+
+Optionally, set different warehouse delivery processes for a shipping method by configuring
+different :doc:`routes <daily_operations/use_routes>` for it.
+
+.. example::
+   Configuring multiple routes per shipping method is helpful for adjusting warehouse delivery
+   processes based on:
+
+   - speed (e.g., use :doc:`one-step delivery <daily_operations/receipts_delivery_one_step>` for
+     express shipping, or :doc:`two-step <daily_operations/receipts_delivery_two_steps>` for
+     standard shipping).
+   - international shipping (e.g., use :doc:`three-step delivery
```

Comment:
```suggestion
   - international shipping (e.g. use :doc:`three-step delivery
```
no need for the comma, since there is only one example in the parenthetical